packages/lab/src/static-list/StaticList.tsx

```diff
@@ -0,0 +1,41 @@
+import { makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import { clsx } from "clsx";
+import {
+  type ComponentPropsWithoutRef,
+  type ReactElement,
+  type ReactNode,
+  forwardRef,
+} from "react";
+
+import staticListCss from "./StaticList.css";
+
+const withBaseName = makePrefixer("saltStaticList");
+
+export interface StaticListProps extends ComponentPropsWithoutRef<"ul"> {
+  /**
+   * Should be one or more components
```

Comment:
could have a more descriptive

The list items to be rendered within the StaticList.

Reply:
What's design recommendation when list is empty?

Reply:
There is no example for this. I think it will depend on the use case?

Reply:
Very good question.A container element for list <ul> or <ol> cannot not exist without the list items. So, I don't know. Let me find out or ask Jake